test/unit/api/stub.test.js

```diff
@@ -8,7 +8,7 @@
 const tap = require('tap')
 const API = require('../../../stub_api')
 
-const EXPECTED_API_COUNT = 29
+const EXPECTED_API_COUNT = 30
```

Comment:
lol. gotta love a random test like this 🤦🏻

Reply:
Yeah, wonder why the number of API methods is so important that its count requires a test.